core/src/scss/utilities/mixins/index.scss

```diff
@@ -13,11 +13,12 @@
 'breakpoint/index',
 'display/index',
 'flex/index',
+'focus',
```

Comment:
Perhaps make this consistent, ie, have the 2 new focus mixins in its own folder with a roll up index file?

Reply:
Thank you, @yvonnetangsu, I was really wanting advice on how to add these. I'll put them in separate folders. 
How do you suggest I name them? Right now they're
_button-focus-override.scss, and
_focus-visible.scss

I think the the name "_focus-visible.scss" works?
I went through a bunch of names for  "_button-focus-override". But, that name does accurately describe its function. 😁

Reply:
Okay, I mis-read. 
lol. 
I believe that SASS automatically looks in  `index.scss` if you just specify the directory name. That's why I didn't need to specify `focus/index`. 
I will make it more explicit! 😁